src/Microsoft.Framework.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -34,7 +34,7 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
             _resourcesRelativePath = localizationOptions.Value.ResourcesPath ?? string.Empty;
             if (!string.IsNullOrEmpty(_resourcesRelativePath))
             {
-                _resourcesRelativePath = _resourcesRelativePath.Replace("/", ".") + ".";
+                _resourcesRelativePath = _resourcesRelativePath.Replace("/", ".").Replace('\\', '.') + ".";
```

Comment:
Profiling might surprise you. We don't make performance changes without measurements to back them up.